routes/utils.js

```diff
@@ -1,10 +1,31 @@
+const jwt = require('jsonwebtoken');
+const { JWT_SECRET } = process.env;
+
+
 const requireUser = (req, res, next) => {
```

Comment:
REQUIRED 
- Some of this functionality  (the bulk of it, the token verify etc) should be added to a middleware `req.use` in the `routes/index.js` file before all API routers get attached.
- Then this function should just check if there is a `req.user` defined.

Reply:
Edited to meet this criteria